docs/docs/reference/policy-groups.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Policy Groups
+---
+
+This feature allow operators to group related policies into one single entity that can be reused across the organization.
+With Policy Groups, materials and policies can be enforced in Chainloop contracts with little or no effort.
+
+For example, they might want to create a "SBOM quality" group with some SBOM-related policies. The policy groups can be defined this way:
+```yaml
+# sbom-quality.yaml
+apiVersion: workflowcontract.chainloop.dev/v1
+kind: PolicyGroup
+metadata:
+  name: sbom-quality
+  description: This policy group applies a number of SBOM-related policies
+  annotations:
+    category: SBOM
+spec:
+  inputs:
+    - name: bannedlicenses
```

Comment:
can you use the `-` in the examples? :) 

```
banned-licenses
```

Reply:
ok, it seems it's not valid, can we use camelcase? would the validations support that? I remember seeing some dns validation

Reply:
Yes, currently it uses dns validation. We'll need to remove it and provide a custom validation instead.

Reply:
For now I'll put it in camelCase